modules/statics/test/unit/coins.ts

```diff
@@ -30,7 +30,17 @@ coins.forEach((coin, coinName) => {
 
     if (!coin.isToken) {
       it(`has expected network type`, function () {
-        coin.network.type.should.eql(coin.name === coin.family ? NetworkType.MAINNET : NetworkType.TESTNET);
+        if (coin.family !== CoinFamily.FIAT) {
+          coin.network.type.should.eql(coin.name === coin.family ? NetworkType.MAINNET : NetworkType.TESTNET);
+        } else {
+          if (coin.name.startsWith('fiat')) {
+            coin.network.type.should.eql(NetworkType.MAINNET);
+          } else if (coin.name.startsWith('tfiat')) {
+            coin.network.type.should.eql(NetworkType.TESTNET);
+          } else {
+            throw new Error(`Ivalid coin name: ${coin.name} should start with fiat or with tfiat`);
```

Comment:
I am indifferent to this being an Error over an assert that always fails, but will let SDK team make final decision.